src/scripts/main.js

```diff
@@ -1 +1,64 @@
 'use strict';
+
+let right;
+let left;
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('mousedown', (click) => {
+    if (click.button === 0) {
+      setTimeout(() => {
+        resolve();
+      }, 3000);
+      left = true;
+    }
+  });
+});
+
+firstPromise.then(() => {
+  const box1 = document.createElement('div');
+
+  document.body.appendChild(box1);
+  box1.textContent = 'First promise was resolved';
```

Comment:
you can create a separate function for displaying the notification and reuse it for each promise